### DISCUSSION TITLE
Statically build vitest UI

### DISCUSSION THREAD

**Discussion by TorbjornHoltmon** [vitest-dev/vitest#2373] [Ideas]
~~Vitest UI is great, and I personally could have use of statically building the UI and publish them to internal sites, in order to display test results.~~

~~Could it be a possible feature?~~
~~How many others see any use for it?~~

The idea is already on the table with #1716 as pointed out by sheremeta-va.

**Comment by sheremet-va** [comment]
I think you mean this? https://github.com/vitest-dev/vitest/issues/1716

I also like this idea
